src/app/clusters/level-control/level-control.cpp

```diff
@@ -481,19 +483,20 @@ bool emberAfLevelControlClusterStepWithOnOffCallback(app::CommandHandler * comma
     auto & stepSize       = commandData.stepSize;
     auto & transitionTime = commandData.transitionTime;
 
-    emberAfLevelControlClusterPrintln("%pSTEP_WITH_ON_OFF %x %x %2x", "RX level-control:", stepMode, stepSize, transitionTime);
+    emberAfLevelControlClusterPrintln("%pSTEP_WITH_ON_OFF %x %x %2x", "RX level-control:", stepMode, stepSize,
+                                      transitionTime.Value());
     stepHandler(commandPath.mEndpointId, Commands::StepWithOnOff::Id, stepMode, stepSize, transitionTime, 0xFF, 0xFF);
```

Comment:
This will pop up as a merge conflict with #21312 .  Per cluster spec, the `OptionsMask` and `OptionsOverride` are not optional (and this PR captured that as well in `general.xml`); those values need to be pulled out of `commandData` and passed to `stepHandler`.  Same for the other variants of `*WithOnOffCallback`